src/AzureFunctions/deploy.msbuild

```diff
@@ -0,0 +1,94 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Deploy" ToolsVersion="4.0">
+	<PropertyGroup>
+		<GXDeployFileProject>$([System.IO.Path]::GetFullPath('$(DeployFullPath)\..\..\..\..\web\$(ProjectName).gxdproj'))</GXDeployFileProject>
+	</PropertyGroup>
+	
+	<Import Project="$(GX_PROGRAM_DIR)\GeneXus.AzureDeploy.targets"/>
+	<Import Project="$(GXDeployFileProject)"/>
+	
+	<PropertyGroup Condition="'$(GENERATOR)' == '.NET Core'">
+		<UploadSource>$(DeployFullPath)</UploadSource>
+		<DeploymentTargetPath>$(GX_PROGRAM_DIR)\DeploymentTargets\AzureFunctions\AzFunctionFiles</DeploymentTargetPath>
+	</PropertyGroup>
+	
+	<ItemGroup>
+		<AllObjects Include="$(ObjectNames.ToLower().Replace(`procedure:`, ``).Split(`;`))"/>
+	</ItemGroup>
+
+	<ItemGroup>
+		<MSDependencies Include = "$(DeploymentTargetPath)\bin\Microsoft.Azure.WebJobs.Extensions.Storage.dll"/>
+		<MSDependencies Include = "$(DeploymentTargetPath)\bin\NCrontab.Signed.dll"/>
+		<MSDependencies Include = "$(DeploymentTargetPath)\bin\Microsoft.WindowsAzure.Storage.dll"/>
+	</ItemGroup>
+	
+	<ItemGroup>
+		<Handler Include = "$(DeploymentTargetPath)\bin\AzBlobTriggerHandler.dll" Condition = "'$(AZURE_FUNCTIONS_TRIGGER_TYPE)' == 'BlobStorage'" />
+		<Handler Include = "$(DeploymentTargetPath)\bin\AzQueueTriggerHandler.dll" Condition = "'$(AZURE_FUNCTIONS_TRIGGER_TYPE)' == 'Queue'" />
+		<Handler Include = "$(DeploymentTargetPath)\bin\AzTimerTriggerHandler.dll" Condition = "'$(AZURE_FUNCTIONS_TRIGGER_TYPE)' == 'Timer'" />
+	</ItemGroup>
+
+	<Target Name="GenerateConfigs" >
+		<GenerateAzFunctionConfigs Condition="%(Object.Type) == 'Procedure' AND %(Object.selected) == 'true'"
```

Comment:
I think (maybe I'm wrong), that this will generate a Config for every Procedure (whether or not is Main Program).

Reply:
Ah, ok, "selected" attribute may filter not Main Procedures.